### PR TITLE
Add contributing section

### PR DESCRIPTION
**Pull request by dviererbe**

### CODE REVIEW

**Comment by setharnold** [comment]
I like it, +1, thanks

File: README.md [hunk 1]
```diff
@@ -932,3 +932,15 @@ we can do two things:
...
 * [Security team MIR JIRA board](https://warthogs.atlassian.net/jira/software/c/projects/SEC/boards/594)
+
+## Contributing
+
+We welcome everyone who wants to improve the Ubuntu documentation! Whether you've found a typo, have a suggestion for improving existing content, or want to add new content, we'd love to hear from you.
```
Comment (slyon) [inline README.md:938]: nitpick: I wonder if this should read "MIR documentation" or "Ubuntu MIR documentation"?

File: README.md [hunk 2]
```diff
@@ -932,3 +932,15 @@ we can do two things:
...
+
+To contribute, simply submit a pull request with your changes or create an issue. Please also attend the weekly MIR Team meeting (Tuesdays 15:30-16:00 GMT) on our [IRC](https://wiki.ubuntu.com/IRC/ChannelList) in the [#ubuntu-meeting](https://wiki.ubuntu.com/BeginnersTeam/Meetings) channel to discuss your issue with the MIR Team.
+
+### Code of Conduct
+
+Please note that we have a code of conduct in place for all contributors to this repository. By contributing, you agree to abide by the [Ubuntu Code of Conduct](https://launchpad.net/codeofconduct)
+
+Thank you for considering contributing to the Ubuntu documentation!
```
Comment (slyon) [inline README.md:946]:
```suggestion
Thank you for considering contributing to Ubuntu!
```

**Review by slyon — APPROVED** [review]
+1

Just a few tiny inline comments for consideration.

File: README.md [hunk 3]
```diff
@@ -932,3 +932,15 @@ we can do two things:
...
+
+## Contributing
+
+We welcome everyone who wants to improve the Ubuntu documentation! Whether you've found a typo, have a suggestion for improving existing content, or want to add new content, we'd love to hear from you.
+
+To contribute, simply submit a pull request with your changes or create an issue. Please also attend the weekly MIR Team meeting (Tuesdays 15:30-16:00 GMT) on our [IRC](https://wiki.ubuntu.com/IRC/ChannelList) in the [#ubuntu-meeting](https://wiki.ubuntu.com/BeginnersTeam/Meetings) channel to discuss your issue with the MIR Team.
+
+### Code of Conduct
+
+Please note that we have a code of conduct in place for all contributors to this repository. By contributing, you agree to abide by the [Ubuntu Code of Conduct](https://launchpad.net/codeofconduct)
```
Comment (didrocks) [inline README.md:944]: Nitpick: missing finale dot

**Review by didrocks — APPROVED** [review]
Thanks for the additional section. Just one nitpick, otherwise, looks good!

File: README.md (hunk 3 above)
Comment (dviererbe) [reply README.md]: Good spotting! Added it.

File: README.md (hunk 1 above)
Comment (dviererbe) [reply README.md]: I changed it to "Ubuntu MIR documentation". I think when we want to be more specific than we we should name the whole context.

**Pull request merged**